loop/e2e-test/install.sh

```diff
@@ -0,0 +1,27 @@
+
+
+#!/bin/bash
+echo
+echo Downloading latest Nightly...
+echo  
+DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+
+# OSX
+rm -rf $DIR'/LatestNightly.dmg'
+
+
+LATEST_DMG=$(curl -s ftp://ftp.mozilla.org/pub/mozilla.org/firefox/nightly/latest-mozilla-central/ | fgrep en-US.mac.dmg | awk '{print $9}'
+)
+curl -# -C - -o $DIR'/LatestNightly.dmg' "ftp://ftp.mozilla.org/pub/mozilla.org/firefox/nightly/latest-mozilla-central/$LATEST_DMG"
+open $DIR'/LatestNightly.dmg'
+
+# TODO: Download and install latest Firefox browser for Ubuntu.
+# wget https://ftp.mozilla.org/pub/mozilla.org/firefox/releases/latest/linux-x86_64/en-US/firefox-38.0.5.tar.bz2
+# tar -xjvf *.bz2 -C /home/ubuntu
+
+echo
+echo Waiting 20 seconds for disk image to be mounted...
+echo
+sleep 20
```

Comment:
This also marginally frightens me, but I don't know of an alternative (apart from 25, 30, 35, 40, ... seconds).

Unless we somehow can automatically install Firefox via Vagrant/Puppet, but then we're probably installing it everywhere when we probably only need it 20% where.